src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -147,7 +147,7 @@ class DatagridCell extends React.Component {
       selected,
       clickable,
       detached,
-      first: typeof context.onSelectRow !== 'function' && aria.colIndex === 1,
+      first: (typeof context.onSelectRow !== 'function' || !column.frozen) && aria.colIndex === 1,
```

Comment:
seems weird. Can you explain the logic here?

Reply:
Modifier `first` is used to displaying left border. Normally it should be set to `true` when `aria.colIndex === 1` but we also have frozen columns and inside we have checkbox  for selecting row and column's value. So we want to display left border only when it's first column and there is no checkbox for selecting row. We display checkbox when `typeof onSelectRow === 'function' && frozen` so we want to display left border in other case

Reply:
yes but you need to be sure that there is not other frozen rows

Reply:
Frozen rows? can you explain?

Reply:
If there is frozen cols (sorry not rows), you dont want a left border if the column is not frozen, make sense right?